scrumboard/serializers.py

```diff
@@ -1,14 +1,20 @@
 from rest_framework import serializers
 
-from .models import List, Card
+from .models import Board, Card
 
 
-class ListSerializer(serializers.ModelSerializer):
+class CardSerializer(serializers.ModelSerializer):
 
     class Meta:
-        model = List
-
-class CardSerializer(serializers.ModelSerializer):
+        model = Card
+        fields = '__all__'
 
+class BoardSerializer(serializers.ModelSerializer):
+    cards = CardSerializer(read_only=True, many=True)
     class Meta:
-        model = Card
\ No newline at end of file
+        model = Board
+        fields = ('__all__')
```

Comment:
For the sake of consistency, let's make this `'__all__'` by removing the parenthesis

Reply:
Also, remove all the extra new lines at the end of this file. It's alright to leave just one empty new line at the end of the file.

Reply:
Okay.